marklogic-data-hub-central/ui/e2e/cypress/integration/explore/verifyDataInDatabases.spec.tsx

```diff
@@ -35,7 +35,7 @@ describe("Verify All Data for final/staging databases and non-entity detail page
 
   it("Switch on zero state page and select query parameters for final database", () => {
     cy.waitUntil(() => toolbar.getExploreToolbarIcon()).click();
-    cy.wait(3000);
+    cy.wait(6000);
     entitiesSidebar.toggleAllDataView();
```

Comment:
Just a suggestion, if you think this wait it's needed I'm ok with it, but maybe instead of waiting so many seconds, we can try increasing the timeout for toggleAllDataView() 

toggleAllDataView() {
    return cy.get(`[aria-label="switch-datasource-all-data"]`, {timeout:50000}).scrollIntoView().click();
}

Reply:
Yeahh, for this one actually the issue wasn't finding the all data view toggle. It's because the test is clicking the toggle before all the requests finish so it causes some issues. And I noticed in nightly that some machines are much slower on the requests so the wait needed quite some time. Let me know if there is a better idea for this. Using intercept on the requests also didn't work for me :(

Reply:
Oh ok, makes sense